tests/src/test/java/alluxio/client/fs/io/FileInStreamIntegrationTest.java

```diff
@@ -99,8 +99,8 @@ public FileInStreamIntegrationTest(BlockStoreType blockStoreType) {
           //  local cache opens a local file on every get call, even for 1 byte read,
           //  which makes small reads extremely slow
           .setProperty(PropertyKey.USER_STREAMING_READER_CHUNK_SIZE_BYTES, Constants.KB)
-          .setProperty(PropertyKey.USER_CLIENT_CACHE_SIZE, ImmutableList.of(100 * Constants.MB))
-          .setProperty(PropertyKey.USER_CLIENT_CACHE_DIRS,
+          .setProperty(PropertyKey.WORKER_PAGE_STORE_SIZES, ImmutableList.of(100 * Constants.MB))
```

Comment:
The purpose of this integration test is using a client to hit the worker storage,  so actually it will use these two to initialize a worker with the new paging store.